templates/common/infra/bicep/core/storage/storage-account.bicep

```diff
@@ -2,10 +2,20 @@ param name string
 param location string = resourceGroup().location
 param tags object = {}
 
+@allowed([ 'Hot', 'Cool', 'Premium' ])
+param accessTier string = 'Hot'
 param allowBlobPublicAccess bool = false
+param allowCrossTenantReplication bool = true
+param allowSharedKeyAccess bool = true
 param containers array = []
+param defaultToOAuthAuthentication bool = false
+param deleteRetentionPolicy object = {}
+@allowed([ 'AzureDnsZone', 'Standard' ])
+param dnsEndpointType string = 'Standard'
 param kind string = 'StorageV2'
 param minimumTlsVersion string = 'TLS1_2'
+@allowed([ 'Enabled', 'Disabled' ])
+param publicNetworkAccess string = 'Disabled'
 param sku object = { name: 'Standard_LRS' }
```

Comment:
I'm assuming these default values match the bicep defaults if they previously weren't specified?  Just ensuring their isn't any breaking changes introduced.

Reply:
The only one in question is the publicNetworkAccess, which I've changed to default to '' instead of Disabled, which should be the Azure default (docs are unclear).

I'll commit that and test our storage/swa/func templates.